src/v2/models/AppState.js

```diff
@@ -129,13 +129,31 @@ export default Model.extend({
     return FORMS_FOR_VERIFICATION.includes(currentFormName);
   },
 
+  removeStateHandleObjects(obj, parentObj, objKeyInParent) {
```

Comment:
Has this been moved to underscore wrapper already?

Reply:
I am not sure. Checked [Okta.js](https://github.com/okta/okta-ui/blob/master/packages/courage/src/util/Okta.js), but was not able to find anything similar. 
Please advise, is there any place where I can take a look?